website/www/site/content/en/documentation/io/built-in/webapis.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Web Apis I/O connector"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[Built-in I/O Transforms](/documentation/io/built-in/)
+
+# Web APIs I/O connector
+
+{{< language-switcher java py go >}}
+
+The Beam SDKs include a built-in transform, called RequestResponseIO that can read from and write to Web APIs such as
+REST or gRPC.
+
+## Before you start
+
+{{< paragraph class="language-java" >}}
+To use RequestResponseIO, add the Maven artifact dependency to your `pom.xml` file. See
+[Maven Central](https://central.sonatype.com/artifact/org.apache.beam/beam-sdks-java-io-rrio) for available versions.
+{{< /paragraph >}}
+
+{{< highlight java >}}
+<dependency>
+    <groupId>org.apache.beam</groupId>
+    <artifactId>beam-sdks-java-io-rrio</artifactId>
+    <version>{{< param release_latest >}}</version>
+</dependency>
+{{< /highlight >}}
+
+{{< paragraph class="language-py" >}}
+To use RequestResponseIO, install the Beam SDK by running `pip install apache-beam`
+{{< /paragraph >}}
```

Comment:
Can we add a note that currently there are no python examples on the website (tracked by https://github.com/apache/beam/issues/30422), but an E2E example does exist in the beam repo at https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/requestresponse_it_test.py

Reply:
I removed the distracting go/python placeholders.

Reply:
I'd still recommend having a callout that a Python implementation does exist (plus the example). Its cheap and better than not having it documented at all.